dbdimp.h

```diff
@@ -537,6 +537,7 @@ struct imp_dbh_st {
                                */
     bool use_server_side_prepare;
     bool disable_fallback_for_server_prepare;
+    bool use_multi_statements;
```

Comment:
Any plans to add a test for the new behaviour?

Reply:
I added a new test which changes `$dbh->{mariadb_multi_statements}` at runtime.